docs/source/model_doc/character_bert.rst

```diff
@@ -0,0 +1,114 @@
+.. 
```

Comment:
Your model should also be added in the main README

Reply:
Is the documentation page `https://huggingface.co/transformers/model_doc/character_bert.html` going to be created from the `.rst` file ? I imagine so.

Reply:
Yes, but  I meant the main repo README.